### ISSUE TITLE
Allow to specify raw shards

### ISSUE THREAD

**Issue by KagChi** [Deivu/Indomitable#7]

**Comment by Deivu** [comment]
What kind of raw shards, like the whole sharder will only spawn specific set of shards?

**Comment by KagChi** [comment]
Yeah, im thinking splitting my bot to different machine but. Indomitable doesnt support specifying shards ids.
